<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Parallel Test Suite" duration-ms="116967" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:04:07Z">
    <groups>
    </groups>
    <test name="SignUpTest" duration-ms="91985" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:03:42Z">
      <class name="com.testingfoo.tests.SignUpTest">
        <test-method status="PASS" signature="suiteSetup()[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="suiteSetup" is-config="true" duration-ms="62" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- suiteSetup -->
        <test-method status="PASS" signature="setUpTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="setUpTest" is-config="true" duration-ms="17587" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:02:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Android]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Android Phone]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[dfa3b957]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[8201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="FAIL" signature="signUpTestWithValidCredentials()[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="signUpTestWithValidCredentials" duration-ms="55467" started-at="2019-08-29T04:02:28Z" finished-at="2019-08-29T04:03:23Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testingfoo.pages.HomePage.isOrderNowBtnAvailable(HomePage.java:63)
at com.testingfoo.tests.SignUpTest.signUpTestWithValidCredentials(SignUpTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signUpTestWithValidCredentials -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="afterMethod" is-config="true" duration-ms="17661" started-at="2019-08-29T04:03:24Z" finished-at="2019-08-29T04:03:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="tearDown" is-config="true" duration-ms="760" started-at="2019-08-29T04:03:41Z" finished-at="2019-08-29T04:03:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownAll()[pri:0, instance:com.testingfoo.tests.SignUpTest@118102ee]" name="tearDownAll" is-config="true" duration-ms="1" started-at="2019-08-29T04:04:07Z" finished-at="2019-08-29T04:04:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAll -->
      </class> <!-- com.testingfoo.tests.SignUpTest -->
    </test> <!-- SignUpTest -->
    <test name="LoginTest" duration-ms="116967" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:04:07Z">
      <class name="com.testingfoo.tests.LoginTest">
        <test-method status="PASS" signature="setUpTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testingfoo.tests.LoginTest@4bafe935]" name="setUpTest" is-config="true" duration-ms="107871" started-at="2019-08-29T04:02:10Z" finished-at="2019-08-29T04:03:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Android]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Android Phone]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[dfa3b957]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[8201]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="FAIL" signature="loginTestWithValidCredentials()[pri:0, instance:com.testingfoo.tests.LoginTest@4bafe935]" name="loginTestWithValidCredentials" duration-ms="8953" started-at="2019-08-29T04:03:58Z" finished-at="2019-08-29T04:04:07Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testingfoo.pages.HomePage.clickMenu(HomePage.java:54)
at com.testingfoo.tests.LoginTest.loginTestWithValidCredentials(LoginTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestWithValidCredentials -->
        <test-method status="FAIL" signature="afterMethod()[pri:0, instance:com.testingfoo.tests.LoginTest@4bafe935]" name="afterMethod" is-config="true" duration-ms="72" started-at="2019-08-29T04:04:07Z" finished-at="2019-08-29T04:04:07Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.testingfoo.pages.HomePage.clickMenu(HomePage.java:54)
at com.testingfoo.tests.LoginTest.afterMethod(LoginTest.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.testingfoo.tests.LoginTest@4bafe935]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-08-29T04:04:07Z" finished-at="2019-08-29T04:04:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testingfoo.tests.LoginTest -->
    </test> <!-- LoginTest -->
  </suite> <!-- Parallel Test Suite -->
</testng-results>
